src/uct/ib/rc/accel/rc_mlx5_ep.c

```diff
@@ -11,6 +11,18 @@
 #include <ucs/sys/compiler.h>
 #include <arpa/inet.h> /* For htonl */
 
+#if HAVE_IBV_EXP_DM
+typedef struct uct_rc_mlx5_tm_dm_pack_data {
```

Comment:
maybe just `uct_rc_mlx5_dm_pack_data`? Otherwise (logically) you would need to guard it with `#if IBV_EXP_HW_TM`